generators/chipyard/src/main/scala/iobinders/IOBinders.scala

```diff
@@ -560,12 +560,42 @@ class WithNMITiedOff extends ComposeIOBinder({
   }
 })
 
+// This IOBinder connects the signals exposed by the CanHavePeripheryGCD trait
+// to the top-level ports of the chip. It handles both the 'busy' signal and
+// the optional external clock signal.
 class WithGCDBusyPunchthrough extends OverrideIOBinder({
```

Comment:
Seems like `WithGCDBusy..` is not really accurately named anymore, since there is a optionally a clock as well now.
Maybe `WithGCDIOPunchthrough`?

Reply:
Yep, can you rename this?

Reply:
Done